builtin/providers/ultradns/common.go

```diff
@@ -155,11 +154,9 @@ func mapFromLimit(name string, l udnssdk.ProbeDetailsLimitDTO) map[string]interf
 
 // hashLimits generates a hashcode for a limits block
 func hashLimits(v interface{}) int {
-	var buf bytes.Buffer
 	m := v.(map[string]interface{})
-	buf.WriteString(fmt.Sprintf("%s-", m["name"].(string)))
```

Comment:
well, other things had multiple attributes concatenated together. This was a copy-paste.